pokedex_async_redux/lib/state/action/actions.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:async_redux/async_redux.dart';
+import 'package:pokedex_asyn_redux/state/app_state.dart';
+
+///AppState Loading Action
```

Comment:
add a space and also, comment doesn't give me a clear message on what this action is for 😂